usr/sbin/pkcstok_migrate/pkcstok_migrate.mk

```diff
@@ -1,11 +1,4 @@
 sbin_PROGRAMS += usr/sbin/pkcstok_migrate/pkcstok_migrate
-noinst_HEADERS += misc/mech_types.h
-noinst_HEADERS += usr/lib/common/defs.h
-noinst_HEADERS += usr/lib/common/host_defs.h
-noinst_HEADERS += usr/include/local_types.h
-noinst_HEADERS += usr/lib/common/h_extern.h
-noinst_HEADERS += usr/lib/common/pkcs_utils.h
-noinst_HEADERS += usr/lib/common/pin_prompt.h
```

Comment:
OK, /misc/mech_types.h is removed, so also remove the dependency here. But why removing the other lines? The h-files in usr/lib/common and /usr/include are still there?

Reply:
I am removing all of them because it does not make any sense to add them to noinst_HEADERS here. You would do that only if that directory contains headers that you do not want to install. 
Thefact that the sources include those headers is no reason for adding them to noinst_HEADERS.